usr/lib/ep11_stdll/ep11tok.conf

```diff
@@ -81,6 +81,10 @@
 # and CKA_EXTRACTABLE cannot both be true. The default value of
 # CKA_IBM_PROTKEY_EXTRACTABLE is false.
 #
+# AES-XTS related mechanisms are only available if the PKEY_MODE option is 
+# enabled and additional hardware and firmware prerequisites are met. AES-XTS
```

Comment:
maybe change 'is enabled' to 'is not DISABLED' ? The other 2 values mean enabled, right ?